libs/astx/src/astx/subscript.py

```diff
@@ -121,3 +121,27 @@ def get_struct(self, simplified: bool = False) -> ReprStruct:
         value = self._get_struct_wrapper(simplified)
 
         return self._prepare_struct(key, value, simplified)
+
+
+@public
+@typechecked
+class Ellipsis(Expr):
+    """AST class for Ellipsis expressions."""
+
+    def __init__(
+        self,
+        loc: SourceLocation = NO_SOURCE_LOCATION,
+        parent: Optional[ASTNodes] = None,
+    ) -> None:
+        super().__init__(loc=loc, parent=parent)
+        self.kind = ASTKind.EllipsisKind
+
+    def __str__(self) -> str:
```

Comment:
@nagajaideep I think that for the return from str `Ellipsis` is better than `...`
you can remove `__str__` function definition .. because it will automatically return the name of the class

if `__str__` jus returns the same name as the class, it is not necessary anymore to use `__str__` it is done automatically